### PR TITLE
perltidy: update config and name correctly

### PR DESCRIPTION
**Pull request by cgzones**

### CODE REVIEW

**Comment by sumpfralle** [comment]
Yeah - cleaning up!

These changes are suitable for `stable-2.0`, please rebase them.

Regarding the Makefile target: it seems ambiguous to me - at least I confused it with `lint` for a minute. How about something like `apply-formatting`?

**Comment by cgzones** [comment]
Please consider merging this one too, cause the renaming `perltidyrc` -> `.perltidyrc` applies only to master.

**Comment by sumpfralle** [comment]
> Please consider merging this one too, cause the renaming perltidyrc -> .perltidyrc applies only to master.

I am not sure, what you mean. Shouldn't we rename the configuration file in both branches?

**Comment by cgzones** [comment]
Added inline comments.

> I am not sure, what you mean. Shouldn't we rename the configuration file in both branches?

It is correctly named in `stable-2.0`.

**Comment by sumpfralle** [comment]
> It is correctly named in stable-2.0.

Either you or me suffers from a slight confusion here :)
* https://github.com/munin-monitoring/munin/blob/master/perltidyrc
* https://github.com/munin-monitoring/munin/blob/stable-2.0/perltidyrc

(or we are talking about different things)

**Comment by cgzones** [comment]
My mistake, I got confused somewhere

**Comment by sumpfralle** [comment]
I just asked @steveschnepp (he is the primary reader/writer of the non-plugins code) just now in the IRC channel, whether he likes this direction, since I do not feel fully entitled to merge this proposal (from my point of view it is perfectly fine).

@cgzones: please feel invited to join the IRC channel (`#munin` at irc.oftc.net), if you like. There is not too much going on there, but it is a suitable medium for a quick exchange of thoughts. Just saying :)

File: Makefile [hunk 1]
```diff
@@ -60,6 +60,15 @@ perlcritic:
...
 	perlcritic lib/Munin/Node
 
+.PHONY: apply-formatting
+apply-formatting:
+	# Format munin perl files with the recommend perltidy settings
+	# This is recommend, but NOT mandatory
+	@# select all scipts except munin-check
+	perltidy script/munin-async script/munin-asyncd script/munin-cron.PL script/munin-doc script/munin-httpd script/munin-limits script/munin-node script/munin-node-configure script/munin-run script/munin-update
+	@# format munin libraries
+	find lib/ -type f -exec perltidy {} \;
```
Comment (steveschnepp) [inline Makefile]: I sense a big diff incoming

**Comment by sumpfralle** [comment]
@steveschnepp: just to avoid confusion: you are happy with having this changeset merged into master, correct?
(somehow you picked *reject*)

**Comment by steveschnepp** [comment]
there are some merge conflicts

**Comment by cgzones** [comment]
rebased

**Comment by sumpfralle** [comment]
Thank you!

**Pull request merged**